src/tools/rbd_mirror/instance_watcher/Types.h

```diff
@@ -44,7 +44,7 @@ struct ImagePayloadBase : public PayloadBase {
   std::string peer_mirror_uuid;
   std::string peer_image_id;
 
-  ImagePayloadBase() : PayloadBase() {
+  ImagePayloadBase() : PayloadBase(), global_image_id{""}, peer_mirror_uuid{""}, peer_image_id{""} {
```

Comment:
Right now I don't see what is wrong with current ImagePayloadBase and SyncPayloadBase. std::string does not require explicit initialization. And I am reluctant to change this way just to silence the warning until somebody explains what is actually wrong.

@dillaman Do you have any suggestions?